src/components/ChallengeEditor/index.js

```diff
@@ -1435,6 +1448,7 @@ class ChallengeEditor extends Component {
         )
       }
     </React.Fragment>
+    // const useTask = _.find(metadata.challengeTypes, { id: challenge.typeId, isTask: true })
```

Comment:
Please removed it from code.

Reply:
@vikasrohit it look we need this const useTask, somehow it not working if just use isTask.

Reply:
in Create New Work screen isTask is always false , so that we have to use another const useTask